processingchain/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/Workflow.groovy

```diff
@@ -7,6 +7,157 @@ import org.orbisgis.datamanager.JdbcDataSource
+ * @param indicatorUse List of geoindicator types to compute (default ["LCZ", "URBAN_TYPOLOGY", "TEB"]
+ *                  --> "LCZ" : compute the indicators needed for the LCZ classification (Stewart et Oke, 2012)
+ *                  --> "URBAN TYPOLOGY" : compute the indicators needed for the urban typology classification (Bocher et al., 2017)
+ *                  --> "TEB" : compute the indicators needed for the Town Energy Balance model
+ * @param svfSimplified A boolean indicating whether or not the simplified version of the SVF should be used. This
+ * version is faster since it is based on a simple relationship between ground SVF calculated at RSU scale and
+ * facade density (Bernard et al. 2018).
+ * @param prefixName A prefix used to name the output table (default ""). Could be useful in case the user wants to
+ * investigate the sensibility of the chain to some input parameters
+ * @param mapOfWeights Values that will be used to increase or decrease the weight of an indicator (which are the key
+ * of the map) for the LCZ classification step (default : all values to 1)
+ * @param hLevMin Minimum building level height
+ * @param hLevMax Maximum building level height
+ * @param hThresholdLev2 Threshold on the building height, used to determine the number of levels
+ * @param outputFolder The path of the folder to store the results as geojson files. By default, the files are stored
+ * in the subfolder "results" in the inputFolder
+ *
+ * @return the path of the folder that contains the geojson files
+ *
+ *
+ * References:
+ * --> Bocher, E., Petit, G., Bernard, J., & Palominos, S. (2018). A geoprocessing framework to compute
+ * urban indicators: The MApUCE tools chain. Urban climate, 24, 153-174.
+ * --> Jérémy Bernard, Erwan Bocher, Gwendall Petit, Sylvain Palominos. Sky View Factor Calculation in
+ * Urban Context: Computational Performance and Accuracy Analysis of Two Open and Free GIS Tools. Climate ,
+ * MDPI, 2018, Urban Overheating - Progress on Mitigation Science and Engineering Applications, 6 (3), pp.60.
+ * --> Stewart, Ian D., and Tim R. Oke. "Local climate zones for urban temperature studies." Bulletin of the American
+ * Meteorological Society 93, no. 12 (2012): 1879-1900.
+ *
+ */
+def BBTOPO_V2() {
+    create({
+        title "Create all geoindicators from OSM data"
+        inputs datasource: JdbcDataSource, inputFolder: String, distance: 0,indicatorUse: ["LCZ", "URBAN_TYPOLOGY", "TEB"],
+                svfSimplified:false, prefixName: "",
+                mapOfWeights : ["sky_view_factor" : 1, "aspect_ratio": 1, "building_surface_fraction": 1,
+                                "impervious_surface_fraction" : 1, "pervious_surface_fraction": 1,
+                                "height_of_roughness_elements": 1, "terrain_roughness_class": 1],
+                hLevMin : 3, hLevMax: 15, hThresholdLev2: 10,
+                outputFolder:""
+        outputs outputFolder: String
```

Comment:
do we add a parameter 'save' in order to make the file saved or the ITable created depending on if it is true or false ?

Reply:
Nop because you will have multiple tables in the database so I think it's better to save the results in file

Reply:
ok